README.md

```diff
@@ -6,7 +6,19 @@ version numbers.
 
 This resource is heavily inspired by the [official S3 resource](https://gthub.com/concourse/s3-resource)
 
-## Deploying to Concourse
+## Adding to your pipeline
+
+To use the BOSH Deployment Resource, you must declare it in your pipeline as a resource type:
```

Comment:
Is `BOSH Deployment Resource` correct here? Looks like a copy & paste error.

Reply:
sorry yes copy paste error